### PR TITLE
Create a forum announcement

### PR DESCRIPTION
**Pull request by notriddle**
This thing will be published in a few days, once people get a chance to review it.

### CODE REVIEW

File: _posts/2017-08-25-forum.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* they both use `@mention`-style
+
+Also, the bors-ng forum has reply-by-email turned on,
+so you can reply to someone else's post right in your mail client.
+Discourse tops it off with a specialized "mailing list mode" (accessible in your user settings)
+and registered users can create new topics by emailing the right address (such as <user-support@forum.bors.tech>).
+
+## What about the other ones?
+
+IRC shouldn't be the preferred place to discuss bors-ng any more.
```
Comment (Macarse) [inline _posts/2017-08-25-forum.md]: when you say IRC you are also referring to Riot, right?

**Comment by Macarse** [comment]
👍

File: _posts/2017-08-25-forum.md (hunk 1 above)
Comment (notriddle) [inline _posts/2017-08-25-forum.md]: Yeah.

File: _posts/2017-08-25-forum.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+There's a real, official place to ask questions now.
+It's at [forum.bors.tech](https://forum.bors.tech).
+You can log in with your GitHub account, just like you log into bors itself.
+
+## A little information about Discourse
+
+If you've used [elixirforum.com] or [users.rust-lang.org], you've already used Discourse.
+Have fun!
+
+If not, you might still notice some similarity with GitHub issues.
```
Comment (locks) [inline _posts/2017-08-25-forum.md]: similarities?

File: _posts/2017-08-25-forum.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+layout:     post
+title:      Get help using Bors-NG on the new support forum
+date:       2017-08-25 00:00:00
+summary:    There's a real, official place to ask questions now. It's at forum.bors.tech. You can log in with your GitHub account, just like you log into bors itself.
+categories: announcement
+---
+
+There's a real, official place to ask questions now.
+It's at [forum.bors.tech](https://forum.bors.tech).
```
Comment (locks) [inline _posts/2017-08-25-forum.md]: might as well have this link in the same format as others

Comment (notriddle) [inline _posts/2017-08-25-forum.md]: Done.

File: _posts/2017-08-25-forum.md (hunk 2 above)
Comment (notriddle) [inline _posts/2017-08-25-forum.md]: Done.

**Pull request merged**